projects/app/src/client/ui/ReplicacheProvider.tsx

```
@@ -0,0 +1,169 @@
+                  pushVersion: requestBody.pushVersion,
+                  schemaVersion: requestBody.schemaVersion,
+                });
+
+                return await trpcToReplicache(response);
+              }
+            : undefined,
+          puller: isAuthenticated
+            ? async (requestBody, requestId) => {
+                invariant(requestBody.pullVersion === 1);
+                const cookie = cookieSchema.parse(requestBody.cookie);
+                invariant(typeof requestBody.cookie === `object`);
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)